packages/ember-views/lib/system/build-component-template.js

```diff
@@ -43,6 +44,30 @@ export default function buildComponentTemplate({ component, layout, isAngleBrack
   return { createdElement: !!tagName, block: blockToRender };
 }
 
+function canChangeTypeAfterRender(attrs) {
+  // This permits testing of the unbound type attr behavior outside of IE8.
+  if (attrs.ie8SafeInput) {
+    return false;
+  }
+  var mutableInputTypeTextElement, docFragment;
+  if (!docFragment) {
```

Comment:
When can this be truthy?  Seems like it can only ever be `undefined`.  Why not initialize `docFragment` and `mutableInputTypeTextElement` upon declaration?